### PR TITLE
Add `post exists` command

### PR DESCRIPTION
**Pull request by BhargavBhandari90**
For https://github.com/wp-cli/entity-command/issues/180

### CODE REVIEW

**Comment by schlessera** [comment]
I see that you modeled this after `wp comment exists`. That one is also not easily usable through scripting. I think it should be changed as well.

For BC reasons, you could print a `WP_CLI::success()` message for success (which produces an exit code of `0`) or `WP_CLI::halt()` on failure with an exit code of `1`.

**Comment by schlessera** [comment]
@BhargavBhandari90 Are you still up for working on this? Do you need help with something?

**Comment by BhargavBhandari90** [comment]
Hi @schlessera,

Please check my latest commit. Is this what you wanted? If yes then I will modify comment command and feature tests.

Sorry for the delay.

File: src/Post_Command.php [hunk 1]
```diff
@@ -896,4 +896,38 @@ private function get_tags( $post_id ) {
...
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # The post is exists.
+	 *     $ wp post exists 1
+	 *     Success: Post with ID 1337 exists.
+	 *     $ echo $?
+	 *     0
+	 *
+	 *     # The post is not exists.
```
Comment (swissspidy) [inline src/Post_Command.php]: Correct would be:

> The post does not exist.

Comment (BhargavBhandari90) [reply src/Post_Command.php]: oops!!!! Thanks @swissspidy

File: features/post.feature [hunk 2]
```diff
@@ -8,6 +8,12 @@ Feature: Manage WordPress posts
     Then STDOUT should be a number
     And save STDOUT as {POST_ID}
 
+    When I run `wp post exists {POST_ID}`
+    Then STDOUT should be:
+      """
+    Success: Post with ID {POST_ID} exists.
+      """
+
```
Comment (schlessera) [inline features/post.feature]: The exit code should also be tested here.

Also, you should add a test for the negative case as well.

File: features/post.feature [hunk 3]
```diff
@@ -8,6 +8,17 @@ Feature: Manage WordPress posts
...
     And save STDOUT as {POST_ID}
 
+    When I run `wp post exists {POST_ID}`
+    Then STDOUT should be:
+      """
+    Success: Post with ID {POST_ID} exists.
+      """
+    And the return code should be 0
+
+    When I run `wp post exists 1000`
```
Comment (schlessera) [inline features/post.feature]: As you expect this call to return an error (non-null exit code), you need to use `When I try...`, not `When I run...`

**Comment by BhargavBhandari90** [comment]
@schlessera Requested changes are done. Sorry for not being so late.
